app/routes/users.js

```diff
@@ -28,7 +28,22 @@ module.exports = [{
       return h.response(result).code(200)
     } catch (e) {
       if (e instanceof DuplicateResourceError) {
-        return h.response(e.message).code(409)
+        const messageArray = []
+
+        if (e.message.toLowerCase().includes('username')) {
```

Comment:
This seems a bit flakey. I assume it's difficult to just pass the e.message regardless of content (as you already know it's a duplicate resource error)?